Corona-Warn-App/src/main/java/de/rki/coronawarnapp/risk/changedetection/CombinedRiskLevelChangeDetector.kt

```diff
@@ -50,10 +53,20 @@ class CombinedRiskLevelChangeDetector @Inject constructor(
             }
             .filter { it.size == 2 }
             .onEach { results ->
-                Timber.v("Checking for combined risklevel change.")
+                Timber.v("Checking for low-to-high or high-to-low risklevel change.")
                 results.checkForRiskLevelChanges()
             }
-            .catch { Timber.e(it, "App config change checks failed.") }
+            .catch { Timber.e(it, "RiskLevel checks failed.") }
+            .launchIn(appScope)
+
+        // send notifications for additional HIGH risks after an initial HIGH risk
+        riskLevelStorage.latestAndLastSuccessfulCombinedEwPtRiskLevelResult
+            .map { it.lastSuccessfullyCalculated }
```

Comment:
I gave you the wrong information. You need to use lastCalculated instead (data is overwritten even if it fails, so the available data for risk encounters belongs to the last calculation)

Reply:
Changed it. Thanks!